### PR TITLE
Issue#344 Snack bar added for notifying user for new messages. 

### PR DESCRIPTION
**Pull request by harshitagupta30**
@kunall17 Please check it out and review!

### CODE REVIEW

File: app/src/main/java/com/zulip/android/activities/MessageListFragment.java [hunk 1]
```diff
@@ -154,6 +159,14 @@ public void onScrollStateChanged(RecyclerView recyclerView, int newState) {
     }
 
     @Override
+    public void onViewCreated(View view, @Nullable Bundle savedInstanceState) {
+        super.onViewCreated(view, savedInstanceState);
+        CoordinatorLayout coordinatorLayout = (CoordinatorLayout) getActivity().findViewById(R.id.coordinatorLayout);
+        snackbar = Snackbar.make(coordinatorLayout,"No new messages",Snackbar.LENGTH_SHORT);
```
Comment (kunall17) [inline app/src/main/java/com/zulip/android/activities/MessageListFragment.java]: User displayed string in strings.xml

Comment (kunall17) [inline app/src/main/java/com/zulip/android/activities/MessageListFragment.java]: Format your code!

Comment (kunall17) [inline app/src/main/java/com/zulip/android/activities/MessageListFragment.java]: It would be better to set custom duration of maybe 1 sec!

**Comment by harshitagupta30** [comment]
Hi @kunall17! I have made the changes as per you said! Please review! :)

**Comment by kunall17** [comment]
@harshitagupta30 Squash the 2 commits!

**Comment by harshitagupta30** [comment]
@kunall17 closing this PR facing some merge conflicts issue! will send a new PR.

**Pull request closed without merging**